### PR TITLE
chore(docs): standardize docs manifest parents

### PR DESCRIPTION
**Pull request by CaptainCarpensir**
Make all parent fields display parents up to the top tree structure in the manifest

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]

File: site/content/docs/include/storage.en.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ volumes:
 <span class="parent-field">storage.volumes.</span><a id="volume" href="#volume" class="field">`volume`</a> <span class="type">Map</span>  
 Specify the configuration of a volume.
 
-<span class="parent-field">volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>  
+<span class="parent-field">storage.volumes.volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>
```
Comment (iamhopaul123) [inline site/content/docs/include/storage.en.md]: Can we make this field consistent with [this](https://aws.github.io/copilot-cli/docs/manifest/pipeline/#stages-deployments-name), which seems to make more sense.

File: site/content/docs/manifest/scheduled-job.en.md [hunk 2]
```diff
@@ -179,28 +179,28 @@ volumes:
 <span class="parent-field">storage.volumes.</span><a id="volume" href="#volume" class="field">`volume`</a> <span class="type">Map</span>  
 Specify the configuration of a volume.
 
-<span class="parent-field">volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>  
+<span class="parent-field">storage.volumes.volume.</span><a id="path" href="#path" class="field">`path`</a> <span class="type">String</span>
```
Comment (iamhopaul123) [inline site/content/docs/manifest/scheduled-job.en.md]: same here

File: site/content/docs/manifest/worker-service.en.md [hunk 3]
```diff
@@ -95,13 +95,13 @@ If specified, creates a dead letter queue and a redrive policy which routes mess
 <span class="parent-field">subscribe.</span><a id="subscribe-topics" href="#subscribe-topics" class="field">`topics`</a> <span class="type">Array of `topic`s</span>  
 Contains information about which SNS topics the worker service should subscribe to.
 
-<span class="parent-field">topic.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>  
+<span class="parent-field">subscribe.topics.topic.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>
```
Comment (iamhopaul123) [inline site/content/docs/manifest/worker-service.en.md]: Same for this one

File: site/content/docs/manifest/worker-service.en.md [hunk 4]
```diff
@@ -95,13 +95,13 @@ If specified, creates a dead letter queue and a redrive policy which routes mess
 <span class="parent-field">subscribe.</span><a id="subscribe-topics" href="#subscribe-topics" class="field">`topics`</a> <span class="type">Array of `topic`s</span>  
 Contains information about which SNS topics the worker service should subscribe to.
 
-<span class="parent-field">topic.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>  
+<span class="parent-field">subscribe.topics.</span><a id="topic-name" href="#topic-name" class="field">`name`</a> <span class="type">String</span>
```
Comment (rclinard-amzn) [inline site/content/docs/manifest/worker-service.en.md]: This is inconsistent with:

https://github.com/aws/copilot-cli/blob/87eec019eb3d9c1ecc78fcb20d2bb47759f4b406/site/content/docs/include/publish.en.md?plain=1#L26

I'm not sure which one is "correct", but we should probably stay consistent.

**Review by rclinard-amzn — APPROVED** [review]
Nice work, these are some good changes. 👍

**Pull request merged**